app/models/member.rb

```diff
@@ -60,9 +60,13 @@ def organised_chapters
   end
 
   def received_welcome_for?(subscription)
-    return received_student_welcome_email if subscription.student?
-    return received_coach_welcome_email if subscription.coach?
-    true
+    if subscription.student?
+      received_student_welcome_email
+    elsif subscription.coach?
+      received_coach_welcome_email
+    else
+      true
+    end
```

Comment:
Personally, I don't think this is an objective improvement, but happy to let it go given the rest of the work here!